src/components/common/ChangeQuantity/ChangeQuantity.tsx

```diff
@@ -15,14 +15,14 @@ export interface ChangeQuantityProps {
 const ChangeQuantity = ({ quantity, decreaseQuantity, increaseQuantity }: ChangeQuantityProps) => {
   return (
     <S.ChangeQuantity>
-      <Button size="s" onClick={decreaseQuantity} square={true} isDisabled={quantity === 1} aria-label="minus">
-        <img src={MINUS} />
+      <Button size="s" onClick={decreaseQuantity} isSquare={true} disabled={quantity === 1} aria-label="minus">
+        <img src={MINUS} alt="수량 감소" />
       </Button>
       <S.Quantity>
         <Text weight="m">{quantity}</Text>
       </S.Quantity>
-      <Button size="s" onClick={increaseQuantity} square={true} aria-label="plus">
-        <img src={PLUS} />
+      <Button size="s" onClick={increaseQuantity} isSquare={true} aria-label="plus">
+        <img src={PLUS} alt="수량 증가" />
       </Button>
```

Comment:
true인 `boolean` prop은 이렇게 간단하게 써도 좋겠습니다.

또, 지금은 "plus", "수량 증가" 이렇게 연달아 읽힐 것 같아, 아래와 같이 제안드려보아요.

```suggestion
      <Button size="s" onClick={increaseQuantity} isSquare aria-label="수량 증가">
        <img src={PLUS} alt="" />
      </Button>
```

Reply:
보조기기를 사용하시는 분들의 인터넷 사용 방법이 아마 요소를 클릭하고 html에 적혀있는 설명을 읽어주는 거라고 알고 있었는데요. 이렇게 중복되어 읽힐 줄은 전혀 예상하지 못했습니다. 좋은 피드백 감사합니다!